app/views/reviews/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_tag "/shelters/#{@shelter.id}" do %>
+<%= form_tag "/shelters/#{params[:shelter_id]}", method: :post do %>
```

Comment:
I made a note to ask for clarity on why we had to make this change. I will let you know as soon as I hear back.